.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "8"
+  - "10"
```

Comment:
This probably can be reverted once rebased

Reply:
Rebased it into nonexistence!